### ISSUE TITLE
Support for Chrome 73 new CORS security

### ISSUE THREAD

**Issue by CrazyCoder** [SteamDatabase/BrowserExtension#33]
Many extension features don't work in Chrome Beta 73.0.3683.27 and later. It will stop working on 12 March 2019 in Chrome 73 release as well.

The error in the Console may look like:

```
Cross-Origin Read Blocking (CORB) blocked cross-origin response https://... with MIME type application/json. 
```

See https://www.chromium.org/Home/chromium-security/extension-content-script-fetches .

Workaround: open _chrome://flags_, set **Enable network service** flag to **Disabled**
![network service](https://i.imgur.com/AdLa6q8.png)

**Comment by xPaw** [comment]
This would only affect steamdb.info fetching userdata from Steam store, right? Requests from store to steamdb do have CORS header set.

**Comment by xPaw** [comment]
Published 2.1.0 with the fixes, can you test?

**Comment by CrazyCoder** [comment]
Thanks for the quick fix! It appears to work fine now.
